twisted_fate/deck_coder/deckCoder.py

```diff
@@ -1,8 +1,8 @@
 # Credit to @stelar7, this python implementation is a port of his original javascript implementation
 
 from base64 import b32decode, b32encode
-
-MAX_KNOWN_VERSION = 2
+from io import BytesIO
+MAX_KNOWN_VERSION = 20
```

Comment:
Why change this to 20? Current Max Known Version in RIOT documentation is 2

Reply:
The latest version should be 18, so I changed it to 20.
The previous decoding function does not support the LoR version v2.0.0. The version number is generated by the new decoder I changed to.